opentelemetry-exporter-gcp-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -239,127 +233,129 @@ def _get_metric_descriptor(
         self._metric_descriptors[descriptor_type] = descriptor
         return descriptor
 
-    def _set_start_end_times(self, point_dict, record, metric_descriptor):
-        updated_key = (metric_descriptor.type, record.labels)
-        seconds, nanos = divmod(
-            record.aggregator.last_update_timestamp, NANOS_PER_SECOND
-        )
+    @staticmethod
+    def _to_point(
+        kind: "MetricDescriptor.MetricKind.V",
+        data_point: Union[NumberDataPoint, HistogramDataPoint],
+    ) -> Point:
+        if isinstance(data_point, HistogramDataPoint):
+            mean = (
+                data_point.sum / data_point.count if data_point.count else 0.0
+            )
+            point_value = TypedValue(
+                distribution_value=Distribution(
```

Comment:
Does the python SDK support exemplars?  If so, worth opening an issue, and adding a comment.  Otherwise, feel free to resolve.

Reply:
Not yet unfortunately :(